scripts/cgp_data_loader.py

```diff
@@ -32,26 +31,18 @@ def main(argv=sys.argv[1:]):
                                help="Output actions that would otherwise be performed.")
     dry_run_group.add_argument("--no-dry-run", dest="dry_run", action="store_false",
                                help="Perform the actions.")
-    parser.add_argument("--dss-endpoint", metavar="DSS_ENDPOINT", required=False,
-                        default=DSS_ENDPOINT_DEFAULT,
+    parser.add_argument("--dss-endpoint", metavar="DSS_ENDPOINT", default=DSS_ENDPOINT_DEFAULT,
```

Comment:
I think it would be good to make the endpoint a required parameter.
It would really be bad to load stuff into a different DSS than intended!